src/NServiceBus.AcceptanceTesting/ScenarioContext.cs

```diff
@@ -29,6 +29,8 @@ public void AddTrace(string trace)
 
         internal LogLevel LogLevel { get; set; } = LogLevel.Info;
 
+        internal ConcurrentDictionary<string, int> UnfinishedFailedMessages = new ConcurrentDictionary<string, int>();
```

Comment:
currently not, it will only [fail the test](https://github.com/Particular/NServiceBus/pull/3835/files#diff-4ca9b583e070df8435b9aea8f58708a4R349) if there are messages in this collection. But that could be improved.